rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -15,6 +15,7 @@
 #ifndef RCLCPP__UTILITIES_HPP_
 #define RCLCPP__UTILITIES_HPP_
 
+
```

Comment:
nit: unneeded new line

Reply:
Leftover of when `join()` was here. Reverted.